app/src/main/java/de/westnordost/streetcomplete/quests/way_light/AddWayLight.java

```diff
@@ -0,0 +1,65 @@
+package de.westnordost.streetcomplete.quests.way_light;
+
+import android.os.Bundle;
+import android.text.TextUtils;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddWayLight extends SimpleOverpassQuestType
+{
+	private static final String[] WAYS_WITH_LIGHT = {
+			"motorway", "motorway_link", "trunk", "trunk_link",
+			"primary", "primary_link", "secondary", "secondary_link", "tertiary", "tertiary_link",
+			"unclassified", "residential", "living_street",
+			"pedestrian", "footway",
+			"track", "road",
+			"service",
+			"cycleway",
+			"path"
```

Comment:
For links I think it is very weird to ask. These are usually just small sections in between real roads. Please remove these.
Also, I think motorway and trunk should be removed. While valid tagging, this is too much (analogous to street surface). "lit" is most useful for pedestrian and cyclists, there are none of these on motorways and trunks.